daml-lf/transaction/src/main/scala/com/daml/lf/TransactionNodeStatistics.scala

```diff
@@ -0,0 +1,156 @@
+// Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.lf
+package transaction
+
+object TransactionNodeStatistics {
+
+  /** Container for transaction statistics.
+    *
+    * @param creates number of creates nodes,
+    * @param consumingExercisesByCid number of consuming exercises by contract ID nodes,
+    * @param nonconsumingExercisesByCid number of non-consuming Exercises by contract ID nodes,
+    * @param consumingExercisesByKey number of consuming exercise by contract key nodes,
+    * @param nonconsumingExercisesByKey number of non-consuming exercise by key nodes,
+    * @param fetchesByCid number of fetch by contract ID nodes,
+    * @param fetchesByKey number of fetch by key nodes,
+    * @param lookupsByKey number of lookup by key nodes,
+    * @param rollbacks number of rollback nodes.
```

Comment:
```suggestion
```

Reply:
The IntelliJ signature refactor missed this.  Removed.